code/modules/surgery/bodyparts/helpers.dm

```diff
@@ -7,7 +7,7 @@
 
 	if(!zone)
 		zone = BODY_ZONE_CHEST
-	for(var/obj/item/bodypart/bodypart as anything in bodyparts)
+	for(var/obj/item/bodypart/L as anything in bodyparts) //PARIAH STATION EDIT
```

Comment:
Easiest to just revert these three, since having a full word seems to be more standardised than a single letter in this codebase.
```diff
-	for(var/obj/item/bodypart/L as anything in bodyparts) //PARIAH STATION EDIT
+	for(var/obj/item/bodypart/bodypart as anything in bodyparts)
```